docs/source/user_guide/accessing_results.rst

```diff
@@ -13,19 +13,19 @@ dedicated methods:
 
 .. code:: python
 
-	# Instantiate the solution object
+	Instantiate the solution object
 
 	>>> from ansys.dpf import post
 	>>> from ansys.dpf.post import examples
 	>>> solution = post.load_solution(examples.multishells_rst)
 
-	#  Instantiate a displacement result object
+	Instantiate a displacement result object
 
 	>>> displacement = solution.displacement()
 	>>> # stress, elastic_strain (...) can also be called.
 
-	# Refer to the list below to know which result object can be
-	# instantiated.
+	Refer to the list below to know which result object can be
+	instantiated.
```

Comment:
@akaszynski This is removing the changes made by @RobPasMue who said these parts should be Python comments, so which is it?

Reply:
I may have been wrong @PProfizi. It looks like the boxes are rendered like this

![image](https://user-images.githubusercontent.com/37798125/186368939-5c543fcb-b010-4ebd-ad1a-d92cc31e14fe.png)

If this is the desired output, then its fine.

Reply:
Oh ok, thank you!